test/integration/print_config.cc

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <sstream>
+#include <string>
+
+#include "sdf/sdf.hh"
+
+#include "test_config.h"
+
+/////////////////////////////////////////////////
+TEST(PrintConfig, PreserveIncludes)
+{
+  const std::string modelPath = sdf::testing::TestFile("integration", "model");
+
+  sdf::setFindCallback(
```

Comment:
It's best to use a `ParserConfig` object and call `SetFindCallback` on it instead of changing the global callback function.

Reply:
8b5bd672b50e35f6658619b737c2292a36c79499